sdk/eventhubs/azure-messaging-eventhubs/inc/azure/messaging/eventhubs/event_data_batch.hpp

```diff
@@ -124,19 +108,22 @@ namespace Azure { namespace Messaging { namespace EventHubs {
      *
      * @return uint64_t
      */
-    uint64_t GetMaxBytes() const { return m_maxBytes; }
+    uint64_t GetMaxBytes() const { return m_maxBytes.Value(); }
 
     /** @brief Adds a message to the data batch
      *
      * @param message The message to add to the batch
      */
-    void AddMessage(Azure::Core::Amqp::Models::AmqpMessage message) { AddAmqpMessage(message); }
+    bool AddMessage(Azure::Core::Amqp::Models::AmqpMessage message)
+    {
+      return AddAmqpMessage(message);
+    }
 
     /** @brief Adds a message to the data batch
      *
      * @param message The message to add to the batch
      */
-    void AddMessage(Azure::Messaging::EventHubs::Models::EventData& message);
+    bool AddMessage(Azure::Messaging::EventHubs::Models::EventData const& message);
```

Comment:
What does the returned true/false value mean? Success/fail?

Add `@return` doc comment to explain the return value.